samples/Grover/Grover.qs

```diff
@@ -0,0 +1,77 @@
+﻿namespace GroverSample {
```

Comment:
A lot of the code in the Grover sample is lifted from Microsoft's sample - I think we should provide a link to it somewhere in the docs.

Reply:
Got it!